[0001].[2021.04.03.01.23.48.722377].SRV1.INF:Main.py.<module>.304:Main Program Started
[0002].[2021.04.03.01.23.48.722377].SRV1.ERR:Main.py.<module>.307:Artificial Error Message for testing purpose
[0003].[2021.04.03.01.23.48.723377].SRV1.INF:Main.py.<module>.308:Command Line iniFileName=default.ini
[0004].[2021.04.03.01.23.48.723377].SRV1.INF:Main.py.<module>.310:Command Line VerboseMode=None
[0005].[2021.04.03.01.23.48.724378].SRV1.INF:Main.py.<module>.313:AppSpoManager.DebugVerboseModeOn=True
[0006].[2021.04.03.01.23.48.724378].SRV1.INF:Main.py.<module>.314:AppSpoManager.DebugInfoMessages=True
[0007].[2021.04.03.01.23.48.725379].SRV1.INF:Main.py.<module>.315:AppSpoManager.DebugErrorMessages=True
[0008].[2021.04.03.01.23.48.725379].SRV1.INF:Main.py.<module>.316:AppSpoManager.DebugTargetConsole=False
[0009].[2021.04.03.01.23.48.726379].SRV1.INF:Main.py.<module>.317:AppSpoManager.DebugTargetFile=True
[0010].[2021.04.03.01.23.48.726379].SRV1.INF:Main.py.<module>.318:AppSpoManager.DebugTargetFilePath=Debugs\Debug1
[0011].[2021.04.03.01.23.48.727379].SRV1.INF:Main.py.<module>.319:AppSpoManager.DebugLinesPerFile=1000
[0012].[2021.04.03.01.23.48.727379].SRV1.INF:Main.py.<module>.320:AppSpoManager.AppRunFolder=c:\Apps\PythonTry1
[0013].[2021.04.03.01.23.48.728380].SRV1.INF:Main.py.<module>.343:ModuleFileNameOnly=Main.py
[0014].[2021.04.03.01.23.48.728380].SRV1.INF:Main.py.<module>.351:File=c:\Apps\PythonTry1\Main.py
[0015].[2021.04.03.01.23.48.729381].SRV1.INF:Main.py.<module>.352:FileDirName=c:\Apps\PythonTry1
[0016].[2021.04.03.01.23.48.729381].SRV1.INF:Main.py.<module>.353:FileAbsPath=c:\Apps\PythonTry1\Main.py
[0017].[2021.04.03.01.23.48.730382].SRV1.INF:Main.py.<module>.356:ModuleName1=__main__
[0018].[2021.04.03.01.23.48.730382].SRV1.INF:Main.py.<module>.357:ModuleName2=Main
[0019].[2021.04.03.01.23.48.731383].SRV1.INF:Main.py.<module>.366:Main Program Started
[0020].[2021.04.03.01.23.48.731383].SRV1.INF:Main.py.<module>.367:Python Executable Path=C:\Program Files\Python39\python.exe
[0021].[2021.04.03.01.23.48.732383].SRV1.INF:Main.py.<module>.368:Python Interpreter Version=3.9.2 (tags/v3.9.2:1a79785, Feb 19 2021, 13:44:55) [MSC v.1928 64 bit (AMD64)]
[0022].[2021.04.03.01.23.48.732383].SRV1.INF:Main.py.<module>.371:Sys.Path=['c:\\Apps\\PythonTry1', 'C:\\Program Files\\Python39\\python39.zip', 'C:\\Program Files\\Python39\\DLLs', 'C:\\Program Files\\Python39\\lib', 'C:\\Program Files\\Python39', 'C:\\Users\\user2\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\Program Files\\Python39\\lib\\site-packages']
[0023].[2021.04.03.01.23.48.761914].SRV1.INF:Main.py.<module>.418:Showing MainWindow
[0024].[2021.04.03.01.23.50.207411].SRV1.INF:Main.py.actionDebugTest1Handler.154:192.168.0.1, KAT1.ODA1, KORÝDOR DUVAR
192.168.0.2, KAT2.ODA2, TAVAN HOPARLÖR
192.168.0.3, KAT2.ODA2, KAPI_ÖNÜ_ANANS
[0025].[2021.04.03.01.23.50.208413].SRV1.INF:Main.py.actionDebugTest1Handler.156:192.168.0.1, KAT1.ODA1, KORÝDOR DUVAR
[0026].[2021.04.03.01.23.50.209412].SRV1.INF:Main.py.actionDebugTest1Handler.157:192.168.0.2, KAT2.ODA2, TAVAN HOPARLÖR
[0027].[2021.04.03.01.23.50.211415].SRV1.INF:Main.py.actionDebugTest1Handler.158:192.168.0.3, KAT2.ODA2, KAPI_ÖNÜ_ANANS
